### ISSUE TITLE
Blinds not stopping

### ISSUE THREAD

**Issue by PhilPearson** [anthonywebb/homebridge-cbus#66]
Can anyone confirm that their blinds/shutter control can stop during movement?
I've deployed this at a friends house who is now saying that they cannot stop the blinds midway. The only control they have is fully open or close.
Control from the cbus button panel works fine.

**Comment by DarylMc** [comment]
@PhilPearson 
I haven't tried shutter control but if you hold the button can you select a position with the slider like a dimmer?

**Comment by PhilPearson** [comment]
Yeah which doesn't work with the shutter devices as the hardware doesn't have a sense of position.
I've got a single channel shutter coming on Monday I'll be able to do more testing then.

**Comment by jensenoz** [comment]
Hi, I noticed that the last posting for this bug was August of last year.  I'm new to this whole game and noticed that this bug still exists.  Any movement on it??

**Comment by peterconn** [comment]
@jensenoz 

With the shutter button you can’t stop the blinds this is correct. You can however hold the shutter button for 1 sec and this will display a dimmer slider. From there you can set your percentage of where the blind will sit.

**Comment by AF0013** [comment]
The 'Dimming" feature works fine with my shutters, however, they must be set to 95% to be 'open' otherwise, the blind rolls up too far and will eventually break if the blind is set to 100% open too often ! Is there any way to add a maximum open % within the config file? Otherwise ill have to do it with automations and dummy switches etc .
Thanks

**Comment by JasonY00** [comment]
Sounds like you need to set your limit switches on your shutter. No matter what your request From homebridge-CBus to the shutter is, the inbuilt limit switches should stop the shutter opening/closing beyond their maximum limits even if power is still applied from the CBus shutter relay.

Note that if the c-bus shutter relay time is set correctly, i.e. set it to almost exactly the time it takes for the shutter to open or close, you can just say "Hey Siri, Set the blind to 50%" and it will go to half way and stop. You can chose any percentage that you want and it will go to that level where 0% is closed and 100% is open. Your shutter relay should be defined as a ":shutter" type in homebridge config.json file.

Cheers

Jason
